### DISCUSSION TITLE
目前 `gpu-next` 与第三方着色器的兼容性?

### DISCUSSION THREAD

**Discussion by Akariln** [hooke007/mpv_PlayKit#296] [mpv 通用]
佬，我想请问一下：

在不改变第三方着色器代码（只从各项目官方下载着色器文件并直接使用）的情况下，
当下`gpu-next`与以下第三方着色器兼容吗?

- FSRCNNX
- Krig
- Anime4K v4.x
- SSimSuperRes
- SSimDownscaler
- Adaptive Sharpen
- Noise Static

若不兼容，软解时是否可行呢？

**Comment by hooke007** [comment]
1. 与解码无关
2. 目前已知个别a4k异常
3. 改动比较频繁，依然是激进开发中，谁知道将来是不是要重写着色器系统（已经有纯为gpu-next服务的着色器了，不支持vo=gpu），没有肯定回答

**Reply by hooke007** [reply]
当前基本正常
